src/Symfony/Component/Intl/Locales.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Intl;
+
+use Symfony\Component\Intl\Exception\MissingResourceException;
+
+/**
+ * Gives access to locale-related ICU data.
+ *
+ * @author Bernhard Schussek <bschussek@gmail.com>
+ * @author Roland Franssen <franssen.roland@gmail.com>
+ */
+final class Locales extends AbstractStaticBundle
+{
+    /**
+     * @return string[]
+     */
+    public static function getCodes(): array
+    {
+        return self::readEntry(['Locales'], 'meta');
+    }
+
+    /**
+     * @return string[]
+     */
+    public static function getAliases(): array
+    {
+        return self::readEntry(['Aliases'], 'meta');
+    }
+
+    public static function exists(string $code): bool
+    {
+        try {
+            self::readEntry(['Locales', $code]);
+
+            return true;
+        } catch (MissingResourceException $e) {
+            return false;
+        }
+    }
+
+    public static function getName(string $code, string $displayLocale = null): string
```

Comment:
```suggestion
    public static function getName(string $locale, string $displayLocale = null): string
```

This line makes it most obvious: Both parameters are locales. The one the locale we want to get the name of. The second the locale *in which* we want to get the name.